en/docs/administer/key-managers/configure-global-key-manager.md

```diff
@@ -59,23 +62,26 @@ Let's look at a scenario where a single access token generated for an applicatio
 
 3. Start the WSO2 API Manager.
 
-4. Sign in to the WSO2 API Publisher (`https://<hostname>:9443/publisher`) using the super tenant’s admin credentials
+4. [Create a tenant]({{base_path}}/administer/multitenancy/managing-tenants/) (Ex: abc.com)
+
+5. Sign in to the WSO2 API Publisher (`https://<hostname>:9443/publisher`) using the super tenant’s admin credentials
 
-5. [Create an API]({{base_path}}/design/create-api/create-rest-api/create-a-rest-api/) (Eg: SampleAPI)
+6. [Create an API]({{base_path}}/design/create-api/create-rest-api/create-a-rest-api/) (Eg: SampleAPI)
 
     [![Sample API Creation]({{base_path}}/assets/img/administer/global-keymanager/sample-api-creation.png)]({{base_path}}/assets/img/administer/global-keymanager/sample-api-creation.png)
 
-6. Go to **Portal Configurations** from the left menu and click **Subscriptions**, click the **Subscription Availability** dropdown, and select the desired subscription availability option. For this scenario, select **Available to all the tenants** and Click Save.
+    !!! Note
```

Comment:
Lets add this note after step 7

Reply:
Resolved by https://github.com/wso2/docs-apim/pull/7921/commits/a66b668f482b7cb1d1c03f2e2df6bbf92ab0f1ed